### PR TITLE
Streamlining some editor related deps

### PR DESCRIPTION
**Pull request by psfinaki**

### CODE REVIEW

File: FSharp.Editor.sln [hunk 1]
```diff
@@ -42,21 +40,11 @@ Global
 		{67DA0BF3-AAD3-47F4-9EC6-AD8EC532B5A9}.Proto|Any CPU.Build.0 = Proto|Any CPU
 		{67DA0BF3-AAD3-47F4-9EC6-AD8EC532B5A9}.Release|Any CPU.ActiveCfg = Release|Any CPU
 		{67DA0BF3-AAD3-47F4-9EC6-AD8EC532B5A9}.Release|Any CPU.Build.0 = Release|Any CPU
-		{24399E68-9000-4556-BDDD-8D74A9660D28}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
-		{24399E68-9000-4556-BDDD-8D74A9660D28}.Debug|Any CPU.Build.0 = Debug|Any CPU
-		{24399E68-9000-4556-BDDD-8D74A9660D28}.Proto|Any CPU.ActiveCfg = Debug|Any CPU
-		{24399E68-9000-4556-BDDD-8D74A9660D28}.Release|Any CPU.ActiveCfg = Release|Any CPU
-		{24399E68-9000-4556-BDDD-8D74A9660D28}.Release|Any CPU.Build.0 = Release|Any CPU
```
Comment (vzarytovskii) [inline FSharp.Editor.sln:49]: Why?

**Comment by vzarytovskii** [comment]
Could you please write short explaination why these changes are needed?

**Comment by psfinaki** [comment]
@vzarytovskii that was my quick & naive attempt to remove some things from FSharp.Editor.sln to make it a bit more lightweight and to make more clear dependencies overall. But looking at CI, this seems to be harder that I thought.

**Comment by psfinaki** [comment]
Alright that's not worth it

**Pull request closed without merging**